media_driver/linux/common/ddi/media_libva_caps.cpp

```diff
@@ -2668,23 +2668,23 @@ VAStatus MediaLibvaCaps::QuerySurfaceAttributes(
             attribs[i].type = VASurfaceAttribPixelFormat;
             attribs[i].value.type = VAGenericValueTypeInteger;
             attribs[i].flags = VA_SURFACE_ATTRIB_GETTABLE | VA_SURFACE_ATTRIB_SETTABLE;
-            attribs[i].value.value.i = VA_FOURCC_AYUV;
+            attribs[i].value.value.i = VA_FOURCC_XYUV;
```

Comment:
Directly abandoning AYUV in query would lead to issues in frameworks like gstreamer, since it's already implemented and working currently.

Reply:
Ok, I will add one more support option for XYUV.

Reply:
Linjie,
What about  Y410 and Y416?  Any framework still using it?

Reply:
Should be in the same condition IMHO, we'd better add new FORMAT support instead of removing the previous to avoid possible regression from application.

Reply:
Linjie,
With latest version, all previous format is kept, and only new format added. pls help review again.